apps/backoffice-v2/src/common/components/organisms/Header/hooks/useNavbarLogic/useNavbarLogic.tsx

```diff
@@ -0,0 +1,68 @@
+import { useFiltersQuery } from '@/domains/filters/hooks/queries/useFiltersQuery/useFiltersQuery';
+import { useFilterId } from '@/common/hooks/useFilterId/useFilterId';
+import { useCallback, useMemo } from 'react';
+import { Building, Goal, Users } from 'lucide-react';
+import { TRoutes, TRouteWithChildren } from '@/Router/types';
+
+export const useNavbarLogic = () => {
+  const { data: filters } = useFiltersQuery();
+  const filterId = useFilterId();
+  const individualsFilters = useMemo(
+    () => filters?.filter(({ entity }) => entity === 'individuals'),
+    [filters],
+  );
+  const businessesFilters = useMemo(
+    () => filters?.filter(({ entity }) => entity === 'businesses'),
+    [filters],
+  );
+  const navItems = [
+    {
```

Comment:
I think the categories should be added only if needed, i.e. if `businessFilters?.length` and same for individuals. Maybe you should ask Nitzan what he thinks

Reply:
Added a "no items found" fallback.